content/system76-software.md

```diff
@@ -222,6 +228,14 @@ make install
 
 Now log out and use the <u>Extensions</u> to enable the extenstion.
 
+If `Gio.DBusError` presents itself when attempting to enable the extension within the Extensions application, confirm that the `system76-power.service` is running:
+
+```bash
+sudo systemctl status system76-power.service
+```
+If the service isn't running, revisit [these](https://github.com/system76/docs/edit/master/content/system76-software.md#system76-power-gnome-shell-extension-in-fedora) steps.
```

Comment:
Thanks for adding this! Although we'd typically link using the relative path: 

`[these](/articles/system76-software.md#system76-power-gnome-shell-extension-in-fedora)`

However, I'm wondering if this link is necessary since is jumps to a section that will likely still be visible to the reader. Could we move those instructions down here? 

Alternatively we could say:

`If the service isn't running, revisit the above instructions to mask the `power-profiles-deamon.service`.

I can implement either of these suggestions, but let me know if you have any concerns.